test/runtests.jl

```diff
@@ -143,6 +143,10 @@ end
 @test UnitTests.A.B.C in Documenter.Utilities.submodules(UnitTests.A)
 @test UnitTests.A.B.C.D in Documenter.Utilities.submodules(UnitTests.A)
 
+## NavNode tests
+
+include("navnode.jl")
```

Comment:
Yes, keeping it separate sounds fine to me, as does shuffling around `include`s in needed.